CHANGELOG.md

```diff
@@ -1,13 +1,6 @@
 # Changelog
 
-## Unreleased
-
-### Bugfixes
-
-- **Connector Management**
-  - fixed technical user selection
-
-## 2.2.0-RC2
+## Unreleased 2.2.0-RC3
```

Comment:
this is not related to RC3 changes. Kindly put this change note at the top with title - unreleased

Reply:
@manojava-gk updated the changelog